test/skiplist/stm_skiplist.ml

```diff
@@ -0,0 +1,78 @@
+(** Sequential and Parallel model-based tests of ws_deque *)
```

Comment:
`s/ws_deque/skiplist/` or remove the whole comment as the test is internal and the comment doesn't technically add anything.